### ISSUE TITLE
A few issues...

### ISSUE THREAD

**Issue by sleepless-ninja** [formr/Formr#83]
First off, great work! I'm really enjoying Formr, however, there are few things that are not working.

- required_indicator, doesn't work.
- label, doesn't work.

In those two cases I didn't look deeply in Formr's code to figure out what's going on as it was easy to work around them. It could very well be something specific that's not clear in the documentation, but since I tried to get them to work, I have to assume they don't.

- Bug with email.

A stray " shows up in the output of the <input> tag, as in:

`<input type="email" name="email_address" value="" id="email_address" autocorrect="off" autocapitalize="none" "="" placeholder="You email address" required="">`

Notice "="" that's the browser trying to fix the stray ".

This is getting triggered from _attributes() when an attribute is added.

Which begs the question, why are autocapitalize and autocorrect added only after an attribute is added to those fields? (email, password, etc...)

A couple of suggestions:

- Inline css in messages - BIG no no :)

I'm basically having to buffer the output and sanitizing in order to wrap and style them how I see fit. Which brings me to...

- Default field-wrap divs

It's certainly helpful, but it would be nice if we can opt out when needed. Again, it's not critical but a nice to have. :)

Will report more if anything else comes up.

**Comment by timgavin** [comment]
These have been addressed and will be in the next push. Thanks
